indexes/about-self.js

```diff
@@ -40,14 +39,13 @@ module.exports = class AboutSelf extends Plugin {
     )
   }
 
-  processRecord(record, seq) {
+  processRecord(record, seq, pValue) {
+    if (pValue < 0) return
```

Comment:
Could we move this `if` into `onRecord` (indexes/plugin.js)? I'm thinking that since this is ssb-db2, it can assume it knows about SSB msg formats, and I think in all records we have KVTs, right?